spells/unlock_player.js

```diff
@@ -0,0 +1,36 @@
+// Unlocks a player that is targeted with a "locked spell" fx. annoy
+// Takes two params, context = object containing functions to operate on state,
+// payload = data sent from client-side ex username or player-id for the targeted player
```

Comment:
The payload should only be `playerId` (as per the description: https://github.com/VilmaNilsson/VIP2021-game/issues/26)

Reply:
done :)